powersimdata/input/tests/test_grid.py

```diff
@@ -405,10 +405,8 @@ def test_grid_eq_failure_storage(base_texas):
     test_grid = copy.deepcopy(base_texas)
     gencost = {g: 0 for g in test_grid.storage["gencost"].columns}
     gen = {g: 0 for g in test_grid.storage["gen"].columns}
-    test_grid.storage["gencost"] = test_grid.storage["gencost"].append(
-        gencost, ignore_index=True
-    )
-    test_grid.storage["gen"] = test_grid.storage["gen"].append(gen, ignore_index=True)
+    test_grid.storage["gencost"].loc[0, :] = gencost
+    test_grid.storage["gen"].loc[0, :] = gen
```

Comment:
Will this work the same as `pd.concat`?

Reply:
Good point. This assumes that the dataframes are empty, which happens to be true in this case.

Reply:
For some reason I tried with a simple example and observed following:
<img width="415" alt="image" src="https://user-images.githubusercontent.com/52716585/160013674-f6f4702a-eaf3-4a63-864e-b7bb9750f88e.png">

Reply:
Yeah this one works because the data frames are "empty" but initialized with columns, so we can assign a dict to a row. My guess was that we have a convention of creating an empty `grid.storage` and adding to it later, but I could generalize it if that's preferred.

Reply:
Ah, wasn't paying attention to the `dict`, it's the content of one row rather than a multiple rows. That should work.